res/TensorFlowPythonExamples/examples/prelu/__init__.py

```diff
@@ -1,7 +1,5 @@
 import tensorflow as tf
-from tensorflow.compat.v1.keras import layers
 
-model = tf.compat.v1.keras.Sequential()
-model.add(layers.PReLU())
-# TODO Find a way to freeze Keras model for inference
-model.build((1, 1))
+lhs_ = tf.compat.v1.placeholder(dtype=tf.float32, name="input", shape=[1, 4, 4, 3])
+prelu = tf.keras.layers.PReLU(shared_axes=[1, 2])
+op_ = prelu(input_tensor)
```

Comment:
Did you run this example?
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-2-94ea515b8416> in <module>
      1 lhs_ = tf.compat.v1.placeholder(dtype=tf.float32, name="input", shape=[1, 4, 4, 3])
      2 prelu = tf.keras.layers.PReLU(shared_axes=[1, 2])
----> 3 op_ = prelu(input_tensor)

NameError: name 'input_tensor' is not defined

```

Reply:
use ``lhs_`` instead of  ``input_tensor``, or rename ``lhs_`` above as ``input_tensor`` ;-)

Reply:
![image](https://user-images.githubusercontent.com/14926045/83510015-48071800-a507-11ea-823d-a3805e555353.png)

👍

Reply:
op_ is now of Tensor type!